packages/esm-patient-medications-app/src/drug-order-basket-panel/order-basket-item-tile.component.tsx

```diff
@@ -24,70 +24,84 @@ export default function OrderBasketItemTile({ orderBasketItem, onItemClick, onRe
-          <span className={styles.dosageLabel}>
-            {orderBasketItem.dosage} {orderBasketItem.unit?.value}
-          </span>{' '}
-          <span className={styles.dosageInfo}>
-            &mdash; {orderBasketItem.route?.value ? <>{orderBasketItem.route.value} &mdash; </> : null}
-            {orderBasketItem.frequency?.value ? <>{orderBasketItem.frequency.value} &mdash; </> : null}
-            {t('refills', 'Refills').toUpperCase()} {orderBasketItem.numRefills}{' '}
-            {t('quantity', 'Quantity').toUpperCase()}{' '}
-            {`${orderBasketItem.pillsDispensed} ${orderBasketItem.quantityUnits?.value?.toLowerCase() ?? ''}`}
-            {orderBasketItem.patientInstructions && <>&mdash; {orderBasketItem.patientInstructions}</>}
           </span>
-        </span>
-        <br />
-        <span className={styles.label01}>
-          <span className={styles.indicationLabel}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
-          <span className={styles.dosageInfo}>
-            {!!orderBasketItem.indication ? orderBasketItem.indication : <i>{t('none', 'None')}</i>}
+          <br />
+          <span className={styles.label01}>
+            <span className={styles.indicationLabel}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
+            <span className={styles.dosageInfo}>
+              {!!orderBasketItem.indication ? orderBasketItem.indication : <i>{t('none', 'None')}</i>}
+            </span>
+            {!!orderBasketItem.orderError && (
+              <>
+                <br />
+                <span className={styles.orderErrorText}>
+                  <WarningIcon size={16} /> &nbsp;{' '}
+                  <span className={styles.label01}>{t('error', 'Error').toUpperCase()}</span> &nbsp;
+                  {orderBasketItem.orderError.responseBody?.error?.message ?? orderBasketItem.orderError.message}
+                </span>
+              </>
+            )}
           </span>
-          {!!orderBasketItem.orderError && (
-            <>
-              <br />
-              <span className={styles.orderErrorText}>
-                <WarningIcon size={16} /> &nbsp;{' '}
-                <span className={styles.label01}>{t('error', 'Error').toUpperCase()}</span> &nbsp;
-                {orderBasketItem.orderError.responseBody?.error?.message ?? orderBasketItem.orderError.message}
-              </span>
-            </>
-          )}
-        </span>
+        </div>
+        <Button
+          className={styles.removeButton}
+          kind="ghost"
+          hasIconOnly={true}
+          renderIcon={(props: ComponentProps<typeof TrashCanIcon>) => <TrashCanIcon size={16} {...props} />}
+          iconDescription={t('removeFromBasket', 'Remove from basket')}
+          onClick={() => {
+            shouldOnClickBeCalled.current = false;
+            onRemoveClick();
+          }}
+          tooltipPosition="left"
+        />
```

Comment:
Nit: prefer [IconButton](https://react.carbondesignsystem.com/?path=/docs/components-iconbutton--overview) for icon only button variants.